### DISCUSSION TITLE
ClipVision or not Clipvision in IPadpter Advanced

### DISCUSSION THREAD

**Discussion by Creative-comfyUI** [cubiq/ComfyUI_IPAdapter_plus#648] [General]
I would like to understand the role of the clipvision model in the case of Ipadpter Advanced. I try with and without and see no change.

**Comment by cubiq** [comment]
If you use the UNIFIED LOADER, **do not use** the clip vision input. It is optional and should be used only if you use the legacy ipadapter loader!

**Comment by Creative-comfyUI** [comment]
Thank you. 

I make a lot of test with ipadapter with prompt and prompt zero conditioning. I am trying to figure out how the noise is connected to give the image we want.  What I understand is the style or composition is applied to the base image genrerated by the model and then it is associated to the noise coming from the prompt. How the combination of the noise is done in a way that we have the right result ? That's the point I don't understand.

Are we able to create a node that it controls the noise injection from the prompt in maner that the result of the clip prompt is divided into section that we can control the noise associated with the words to influence the result. Do you think this kind of thing is possible?  At the moment much of the control is on the model rather than the conditioning.
